python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -249,17 +274,10 @@ def launch_run(self, context: LaunchRunContext) -> None:
         # Run a task using the same network configuration as this processes's
         # task.
         response = self.ecs.run_task(
-            taskDefinition=task_definition,
-            cluster=metadata.cluster,
-            overrides=overrides,
-            networkConfiguration={
-                "awsvpcConfiguration": {
-                    "subnets": metadata.subnets,
-                    "assignPublicIp": metadata.assign_public_ip,
-                    "securityGroups": metadata.security_groups,
-                }
-            },
+            taskDefinition=task_config.task_definition,
             launchType="FARGATE",
+            overrides=overrides,
+            **task_config.run_task_kwargs,
```

Comment:
Don't we need to munge `task_config.run_task_kwargs` to not include `taskDefinition`, `launchType`, and `overrides`? Otherwise, we can end up in a situation where the same kwarg is passed to the function twice.

```py
>>> def foo(**kwargs):
...   pass
... 
>>> kwargs = {"bar": 1}
>>> foo(**kwargs)
>>> foo(bar=2, **kwargs)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: foo() got multiple values for keyword argument 'bar'
```